tests/ref/html/bibliography-no-title.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <p role="doc-biblioref"><a id="loc-1" href="#loc-2">[1]</a></p>
```

Comment:
Is it intentional that `role="doc-biblioref"` is attached to the `p` tag and not the `a` tag?

Reply:
No, I didn't notice that. That's an unfortunate interaction of different mechanisms. :/

Reply:
Fixed.